src/components/RecentTickets.js

```diff
@@ -28,7 +31,7 @@ export default class RecentTickets extends Component {
         {this.state.lastResponse ? (
           <div>
             <h3>Recent Tickets:</h3>
-            <table className="ui celled padded table">
+            <table class="ui unstackable celled padded table">
```

Comment:
minor design specification. When zoomed in on browser (or on a mobile/ tablet) a semantic ui will by default stack its table to fit the display. I wasnt liking the way it looked in my browser so I made it unstackable. really minor and I dont think will matter in the long run as we havent yet discussed porting this for mobile or tablet.

Reply:
semantic ui references also simply had class when defining an element so I switched that as well

Reply:
Oh sure! `className` is a React paradigm, it's [preferable](https://stackoverflow.com/questions/46989454/class-vs-classname-in-react-16) to use it over `class`. The reason they use `class` is because the framework can also be used for normal HTML/CSS/JS, in which case `class` is the right keyword to use.

Reply:
It'd also be preferable to keep this as `className` just so it's the same as the rest of the codebase.

Reply:
Ill update it now.

Reply:
Thanks!